### PR TITLE
[TA] beta updates

### PR DESCRIPTION
**Pull request by kristapratico**
Related: https://github.com/Azure/azure-sdk-for-python/issues/28288 https://github.com/Azure/azure-sdk-for-python/issues/28274

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
 
 ## 5.3.0b2 (Unreleased)
 
+This version of the client library defaults to the service API version `2022-10-01-preview`.
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:5]: We already mentioned the version updates in the previous beta. Should we keep mentioning it until it changes?

Comment (kristapratico) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:5]: I think it's useful information. I feel like we usually bump the API version when we do a release and this time we're not so it's worth calling out.

**Review by iscai-msft — APPROVED** [review]

File: sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md [hunk 2]
```diff
@@ -2,10 +2,22 @@
...
+
 ### Features Added
 
+- Added `begin_extract_summary` client method to perform extractive summarization on documents.
+- Added `begin_abstract_summary` client method to perform abstractive summarization on documents.
+
 ### Breaking Changes
 
+- Removed models `BaseResolution` and `BooleanResolution`.
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:14]: Is "BaseResolution" a base class that all derived resolutions extends from?

Comment (kristapratico) [reply sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md:14]: Yes, but we don't really expose it and won't ever return it to the user and so architect feedback was to remove it.

**Review by mssfang — APPROVED** [review]

**Pull request merged**